src/AWS.Deploy.CLI/Commands/TypeHints/ElasticBeanstalkVpcCommand.cs

```diff
@@ -0,0 +1,192 @@
+        private readonly IToolInteractiveService _toolInteractiveService;
+        private readonly IOptionSettingHandler _optionSettingHandler;
+
+        public ElasticBeanstalkVpcCommand(IAWSResourceQueryer awsResourceQueryer, IConsoleUtilities consoleUtilities, IToolInteractiveService toolInteractiveService, IOptionSettingHandler optionSettingHandler)
+        {
+            _awsResourceQueryer = awsResourceQueryer;
+            _consoleUtilities = consoleUtilities;
+            _toolInteractiveService = toolInteractiveService;
+            _optionSettingHandler = optionSettingHandler;
+        }
+
+        private async Task<List<Vpc>> GetData()
+        {
+            return await _awsResourceQueryer.GetListOfVpcs();
+        }
+
+        public async Task<TypeHintResourceTable> GetResources(Recommendation recommendation, OptionSettingItem optionSetting)
+        {
+            var vpcs = await GetData();
+            var resourceTable = new TypeHintResourceTable();
+
+            resourceTable.Rows =  vpcs.ToDictionary(x => x.VpcId, x => {
+                var name = x.Tags?.FirstOrDefault(x => x.Key == "Name")?.Value ?? string.Empty;
+                var namePart =
+                    string.IsNullOrEmpty(name)
+                        ? ""
+                        : $" ({name}) ";
+
+                var isDefaultPart =
+                    x.IsDefault
+                        ? " *** Account Default VPC ***"
+                        : "";
+
+                return $"{x.VpcId}{namePart}{isDefaultPart}";
+            }).Select(x => new TypeHintResource(x.Key, x.Value)).ToList();
+
+            return resourceTable;
+        }
+
+        public async Task<object> Execute(Recommendation recommendation, OptionSettingItem optionSetting)
+        {
+            _toolInteractiveService.WriteLine();
+            var useVpcOptionSetting = optionSetting.ChildOptionSettings.First(x => x.Id.Equals("UseVPC"));
+            var useVpcValue = _optionSettingHandler.GetOptionSettingValue<string>(recommendation, useVpcOptionSetting) ?? "false";
+            var useVpcAnswer = _consoleUtilities.AskYesNoQuestion(useVpcOptionSetting.Description, useVpcValue);
+            var useVpc = useVpcAnswer == YesNo.Yes;
+
+            if (!useVpc)
+                return new ElasticBeanstalkVpcTypeHintResponse()
+                {
+                    UseVPC = false
+                };
+
+            var currentVpcTypeHintResponse = optionSetting.GetTypeHintData<ElasticBeanstalkVpcTypeHintResponse>();
+
+            var vpcs = await GetData();
+
+            if (!vpcs.Any())
+            {
+                _toolInteractiveService.WriteLine();
+                _toolInteractiveService.WriteLine("There are no VPCs in the selected account. The only option is to create a new one.");
+                return new ElasticBeanstalkVpcTypeHintResponse
+                {
+                    UseVPC = true,
+                    CreateNew = true
+                };
```

Comment:
Can we apply this logic to set `CreateNew` to true in this case to server mode?

Currently select "Use a VPC" in this case, a user sees a validation error immediately:
![image](https://user-images.githubusercontent.com/1170880/179261845-8dfec7f8-879b-46a2-af5b-45ee4429eb8e.png)

Reply:
Done. Toolkit may not show validations correctly, but I have validated they work in server mode. Working with Chris to make sure this change works in the toolkit.